c2rust-ast-exporter/src/AstExporter.cpp

```diff
@@ -857,6 +857,13 @@ class TranslateASTVisitor final
         abort();
     }
 
+    bool VisitStaticAssertDecl(StaticAssertDecl *SAD) {
+        std::vector<void *> childIds = {SAD->getAssertExpr(), SAD->getMessage()};
+        auto T = QualType(); // don't care about the type
```

Comment:
Does this need to be a separate variable?

Reply:
no, but if I just pass `QualType()` as the 4th argument how do I communicate that we don't care about this parameter?

Reply:
Move the comment to that line?